### PR TITLE
[9.0]-add-mrp_bom_structure_xlsx

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Export BOM Structure to Excel .XLSX
===================================

This module extends the functionality of the MRP capabilities of Odoo,
and to allow you to export the BOM structure to MS Excel .XLSX format.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
Does someone knows why the import of the module 'report_xlsx' fails?

cc @pedrobaeza

**Comment by pedrobaeza** [comment]
Include the repository reporting-engine in the file oca_repositories.txt.

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@tafaRU your review is welcome on this one!

**Comment by JordiBForgeFlow** [comment]
@lreficent can you review?

File: mrp_bom_structure_xlsx/report/bom_structure_xlsx.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<openerp>
+  <data>
```
Comment (LoisRForgeFlow) [inline mrp_bom_structure_xlsx/report/bom_structure_xlsx.xml:5]: correct this.

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot, works fine! good job :+1:

File: mrp_bom_structure_xlsx/report/bom_structure_xlsx.xml (hunk 1 above)
Comment (MiquelRForgeFlow) [reply mrp_bom_structure_xlsx/report/bom_structure_xlsx.xml:5]: This should be done when migrating to v10.

**Pull request merged**